sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationClientTest.java

```diff
@@ -131,6 +134,78 @@ void deleteQueueWithResponse() {
         assertEquals(voidResponse, actual);
     }
 
+    @Test
+    void deleteRule() {
```

Comment:
Do you have record/playback tests for this?

Reply:
adding it